### ISSUE TITLE
pass private SSH key to instance via docker env vars

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida-prerequisites#36]
See https://github.com/CarbonInfinity/AiiDA-on-AWS/issues/2#issue-983696900

@csadorf After Dave, this has also been requested by the guys at Carbon infinity.
Do you prefer if I take it on to move https://github.com/aiidalab/aiidalab-docker-stack-demo/pull/12 here?

**Comment by csadorf** [comment]
@ltalirz Yes, please. I don't have the capacity  to prioritize this right now.
